Index: geos/source/headers/geos/geom/CoordinateSequenceFactory.h
===================================================================
--- geos.orig/source/headers/geos/geom/CoordinateSequenceFactory.h	2011-05-24 15:51:50.000000000 +0200
+++ geos/source/headers/geos/geom/CoordinateSequenceFactory.h	2011-05-24 15:51:53.000000000 +0200
@@ -24,7 +24,7 @@
 #include <geos/export.h>
 #include <vector>
 
-//#include <geos/geom/Coordinate.h>
+#include <geos/geom/Coordinate.h>
 #include <geos/inline.h>
 
 // Forward declarations
