Index: geos/source/headers/geos/planargraph/DirectedEdgeStar.h
===================================================================
--- geos.orig/source/headers/geos/planargraph/DirectedEdgeStar.h	2011-05-25 15:32:28.000000000 +0200
+++ geos/source/headers/geos/planargraph/DirectedEdgeStar.h	2011-05-25 15:33:06.000000000 +0200
@@ -20,6 +20,7 @@
 #include <geos/export.h>
 
 #include <vector>
+#include <cstddef>
 
 // Forward declarations
 namespace geos {
@@ -35,6 +36,8 @@
 namespace geos {
 namespace planargraph { // geos.planargraph
 
+using namespace std;
+
 /// A sorted collection of DirectedEdge which leave a Node in a PlanarGraph.
 class GEOS_DLL DirectedEdgeStar {
 protected:
