Index: geos/source/headers/geos/simplify/DouglasPeuckerLineSimplifier.h
===================================================================
--- geos.orig/source/headers/geos/simplify/DouglasPeuckerLineSimplifier.h	2011-05-25 16:11:31.000000000 +0200
+++ geos/source/headers/geos/simplify/DouglasPeuckerLineSimplifier.h	2011-05-25 16:12:10.000000000 +0200
@@ -23,6 +23,7 @@
 #include <geos/export.h>
 #include <vector>
 #include <memory> // for auto_ptr
+#include <cstddef> 
 
 // Forward declarations
 namespace geos {
@@ -34,6 +35,8 @@
 namespace geos {
 namespace simplify { // geos::simplify
 
+using namespace std;
+
 /** \brief
  * Simplifies a linestring (sequence of points) using
  * the standard Douglas-Peucker algorithm.
