Index: geos/source/headers/geos/index/chain/MonotoneChainBuilder.h
===================================================================
--- geos.orig/source/headers/geos/index/chain/MonotoneChainBuilder.h	2011-05-25 13:12:19.000000000 +0200
+++ geos/source/headers/geos/index/chain/MonotoneChainBuilder.h	2011-05-25 13:12:35.000000000 +0200
@@ -22,6 +22,7 @@
 
 #include <geos/export.h>
 #include <vector>
+#include <cstddef>
 
 // Forward declarations
 namespace geos {
@@ -39,6 +40,8 @@
 namespace index { // geos::index
 namespace chain { // geos::index::chain
 
+using namespace std;
+
 /** \brief
  * Constructs {@link MonotoneChain}s
  * for sequences of {@link Coordinate}s.
