Index: geos/source/headers/geos/operation/valid/SimpleNestedRingTester.h
===================================================================
--- geos.orig/source/headers/geos/operation/valid/SimpleNestedRingTester.h	2011-05-25 15:10:53.000000000 +0200
+++ geos/source/headers/geos/operation/valid/SimpleNestedRingTester.h	2011-05-25 15:11:11.000000000 +0200
@@ -24,6 +24,7 @@
 #include <geos/export.h>
 
 #include <vector>
+#include <cstddef>
 
 // Forward declarations
 namespace geos {
@@ -40,6 +41,8 @@
 namespace operation { // geos::operation
 namespace valid { // geos::operation::valid
 
+using namespace std;
+
 /** \brief
  * Tests whether any of a set of {@link LinearRing}s are
  * nested inside another ring in the set, using a simple O(n^2)
