Index: geos/source/headers/geos/simplify/TaggedLineStringSimplifier.h
===================================================================
--- geos.orig/source/headers/geos/simplify/TaggedLineStringSimplifier.h	2011-05-25 17:59:12.000000000 +0200
+++ geos/source/headers/geos/simplify/TaggedLineStringSimplifier.h	2011-05-25 17:59:30.000000000 +0200
@@ -28,6 +28,7 @@
 #include <geos/export.h>
 #include <vector>
 #include <memory>
+#include <cstddef>
 
 // Forward declarations
 namespace geos {
@@ -48,6 +49,7 @@
 namespace geos {
 namespace simplify { // geos::simplify
 
+using namespace std;
 
 /** \brief
  * Simplifies a TaggedLineString, preserving topology
