Index: geos/source/headers/geos/simplify/TaggedLineString.h
===================================================================
--- geos.orig/source/headers/geos/simplify/TaggedLineString.h	2011-05-25 17:49:06.000000000 +0200
+++ geos/source/headers/geos/simplify/TaggedLineString.h	2011-05-25 17:49:28.000000000 +0200
@@ -29,6 +29,7 @@
 #include <geos/export.h>
 #include <vector>
 #include <memory>
+#include <cstddef>
 
 // Forward declarations
 namespace geos {
@@ -47,6 +48,7 @@
 namespace geos {
 namespace simplify { // geos::simplify
 
+using namespace std;
 
 /** \brief
  *
