diff -c -L /home/sjoerd/.qgis_mobility_generator_cache/source/PyQt-x11-gpl-4.9.6/PyQt-x11-gpl-4.9.6/sip/QtGui/qapplication.sip -L \#\<buffer\ qapplication.sip\> /home/sjoerd/.qgis_mobility_generator_cache/source/PyQt-x11-gpl-4.9.6/PyQt-x11-gpl-4.9.6/sip/QtGui/qapplication.sip /tmp/user/1000/buffer-content-4515MSo
*** a/sip/QtGui/qapplication.sip
--- b/sip/QtGui/qapplication.sip
***************
*** 488,494 ****
      #else
          {0, 0, -1, 8},
      #endif
!         {sipName_QSound, &sipType_QSound, -1, 9},
          {sipName_QItemSelectionModel, &sipType_QItemSelectionModel, -1, 10},
      #if QT_VERSION >= 0x040400
          {sipName_QFileSystemModel, &sipType_QFileSystemModel, -1, 11},
--- 488,494 ----
      #else
          {0, 0, -1, 8},
      #endif
!         {0, 0, -1, 9},
          {sipName_QItemSelectionModel, &sipType_QItemSelectionModel, -1, 10},
      #if QT_VERSION >= 0x040400
          {sipName_QFileSystemModel, &sipType_QFileSystemModel, -1, 11},

Diff finished.  Fri Feb 15 13:26:33 2013
