diff -c -L /home/sjoerd/.qgis_mobility_generator_cache/source/PyQt-x11-gpl-4.9.6/PyQt-x11-gpl-4.9.6/sip/QtGui/qguiapplication.sip -L \#\<buffer\ qguiapplication.sip\> /home/sjoerd/.qgis_mobility_generator_cache/source/PyQt-x11-gpl-4.9.6/PyQt-x11-gpl-4.9.6/sip/QtGui/qguiapplication.sip /tmp/user/1000/buffer-content-4515MZc
*** a/sip/QtGui/qguiapplication.sip
--- b/sip/QtGui/qguiapplication.sip
***************
*** 73,80 ****
      static QClipboard *clipboard();
      static QPalette palette();
      static void setPalette(const QPalette &pal);
-     static Qt::KeyboardModifiers keyboardModifiers();
-     static Qt::KeyboardModifiers queryKeyboardModifiers();
      static Qt::MouseButtons mouseButtons();
      static void setLayoutDirection(Qt::LayoutDirection direction);
      static Qt::LayoutDirection layoutDirection();
--- 73,78 ----

Diff finished.  Fri Feb 15 14:01:58 2013
