diff -c -L /home/sjoerd/.qgis_mobility_generator_cache/source/PyQt-x11-gpl-4.9.6/PyQt-x11-gpl-4.9.6/configure.py -L \#\<buffer\ configure.py\> /home/sjoerd/.qgis_mobility_generator_cache/source/PyQt-x11-gpl-4.9.6/PyQt-x11-gpl-4.9.6/configure.py /tmp/user/1000/buffer-content-4515llt
*** a/configure.py
--- b/configure.py
***************
*** 1539,1545 ****
          else:
              plattag = "WS_MACX"
      else:
-         plattag = "WS_X11"
  
      qt_sip_flags.append("-t")
      qt_sip_flags.append(plattag)
--- 1539,1546 ----
          else:
              plattag = "WS_MACX"
      else:
+         plattag = "WS_NO_WINDOW_MANAGER"
+         pass
  
      qt_sip_flags.append("-t")
      qt_sip_flags.append(plattag)

Diff finished.  Fri Feb 15 10:39:28 2013
