diff -c -L /home/sjoerd/.qgis_mobility_generator_cache/source/PyQt-x11-gpl-4.9.6/PyQt-x11-gpl-4.9.6/sip/QtGui/QtGuimod.sip -L \#\<buffer\ QtGuimod.sip\> /home/sjoerd/.qgis_mobility_generator_cache/source/PyQt-x11-gpl-4.9.6/PyQt-x11-gpl-4.9.6/sip/QtGui/QtGuimod.sip /tmp/user/1000/buffer-content-4515Zcu
*** a/sip/QtGui/QtGuimod.sip
--- b/sip/QtGui/QtGuimod.sip
***************
*** 202,208 ****
  %Include qsizepolicy.sip
  %Include qslider.sip
  %Include qsortfilterproxymodel.sip
! %Include qsound.sip
  %Include qspinbox.sip
  %Include qsplashscreen.sip
  %Include qsplitter.sip
--- 202,208 ----
  %Include qsizepolicy.sip
  %Include qslider.sip
  %Include qsortfilterproxymodel.sip
! 
  %Include qspinbox.sip
  %Include qsplashscreen.sip
  %Include qsplitter.sip

Diff finished.  Fri Feb 15 13:27:23 2013
