diff -c -L /home/sjoerd/.qgis_mobility_generator_cache/source/PyQt-x11-gpl-4.9.6/PyQt-x11-gpl-4.9.6/sip/QtGui/qwindowdefs.sip -L \#\<buffer\ qwindowdefs.sip\> /home/sjoerd/.qgis_mobility_generator_cache/source/PyQt-x11-gpl-4.9.6/PyQt-x11-gpl-4.9.6/sip/QtGui/qwindowdefs.sip /tmp/user/1000/buffer-content-4515xDJ
*** a/sip/QtGui/qwindowdefs.sip
--- b/sip/QtGui/qwindowdefs.sip
***************
*** 26,43 ****
  
  
  typedef QList<QWidget*> QWidgetList;
- %If (WS_X11)
- // X11 specific definitions.
  typedef unsigned long WId;
- %End
- %If (WS_WIN)
- // Windows specific definitions.
- typedef struct HFONT__ *HFONT;
- typedef struct HBITMAP__ *HBITMAP;
- typedef struct HICON__ *HICON;
- typedef HWND WId;
- %End
- %If (WS_MACX)
- // MacOS/X specific definitions.
- typedef long WId;
- %End
--- 26,29 ----

Diff finished.  Fri Feb 15 10:58:02 2013
